client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/InteractionEventForm/AddressForm/AddressForm.tsx

```diff
@@ -0,0 +1,82 @@
+import React, { useContext } from 'react';
+import { Grid } from '@material-ui/core';
+
+import useFormStyles from 'styles/formStyles';
+import FormInput from 'commons/FormInput/FormInput';
+import CircleTextField from 'commons/CircleTextField/CircleTextField';
+import { InteractionEventDialogContext } from 'components/App/Content/InvestigationForm/TabManagement/InteractionsTab/InteractionsEventDialogContext/InteractionsEventDialogContext';
+import InteractionEventDialogData from 'models/Contexts/InteractionEventDialogData';
+import InteractionEventAddressFields from 'components/App/Content/InvestigationForm/TabManagement/InteractionsTab/InteractionsEventDialogContext/InteractionEventAddressFields';
+
+const AddressForm : React.FC<Props> = (props: Props) : JSX.Element => {
+
+    const { removeFloor } = props;
+        
+    const formClasses = useFormStyles();
+    const ctxt = useContext(InteractionEventDialogContext);
+    const { interactionEventDialogData, setInteractionEventDialogData } = ctxt;
+    const { city, houseNumber, street, entrance, floor, neighborhood } = interactionEventDialogData.locationAddress;
+
+    const onChange = (event: React.ChangeEvent<HTMLInputElement | HTMLTextAreaElement>, updatedField: InteractionEventAddressFields) =>
+        setInteractionEventDialogData({...ctxt.interactionEventDialogData as InteractionEventDialogData, 
+            locationAddress: {...interactionEventDialogData.locationAddress, [updatedField]: event.target.value}});
+
+    return (
+        <>
+            <Grid container className={formClasses.formRow}>
+                <Grid item xs={6}>
+                    <FormInput fieldName='עיר'>
```

Comment:
I think that it doesnt have to be common because the componenets arent really related so its possible that one component label will change and the other wont